.github/workflows/mypy.yml

```diff
@@ -10,6 +10,7 @@ on:
       - "Tools/clinic/**"
       - "Tools/cases_generator/**"
       - "Tools/peg_generator/**"
+      - "Tools/wasm/**"
```

Comment:
Nit: please could you sort these existing paths alphabetically? Will help avoid merge conflicts.